src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -44,6 +56,14 @@ public void sortPersonsByLastNameThenFirstNameUsingArraysSortAnonymousComparator
         Person[] persons = getPersons();
 
         // TODO использовать Arrays.sort
+        Arrays.sort(persons, new Comparator<Person>(){
+            @Override
+            public int compare(Person o1, Person o2) {
+
+                int result = o1.getLastName().compareTo(o2.getLastName());
+                return result == 0 ? o1.getFirstName().compareTo(o2.getFirstName()) : result;
```

Comment:
seems like you can use thenCompare()

Reply:
yes, but it is too many lines with anonymous comparator :)